### PR TITLE
Docs: adding GEOMETRY

### PR DESCRIPTION
**Pull request by adrianoamaral**

### CODE REVIEW

File: docs/commands/ft.create.md [hunk 1]
```diff
@@ -51,10 +51,12 @@ after the SCHEMA keyword, declares which fields to index:
 
  - `NUMERIC` - Allows numeric range queries against the value in this attribute. See [query syntax docs](/redisearch/reference/query_syntax) for details on how to use numeric ranges.
 
- - `GEO` - Allows geographic range queries against the value in this attribute. The value of the attribute must be a string containing a longitude (first) and latitude separated by a comma.
+ - `GEO` - Allows radius range queries against the value (point) in this attribute. The value of the attribute must be a string containing a longitude (first) and latitude separated by a comma.
 
  - `VECTOR` - Allows vector similarity queries against the value in this attribute. For more information, see [Vector Fields](/redisearch/reference/vectors).
 
+ - `GEOMETRY`- Allows polygon queries against the value in this attribute. The value of the attribute must follow [WKT notation](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) a list of points representing the polygon edges `POLYGON((x1 y1, x2 y2, ...)` separated by a comma.
```
Comment (oshadmi) [inline docs/commands/ft.create.md:58]:
```suggestion
 - `GEOMETRY`- Allows polygon queries against the value in this attribute. The value of the attribute must follow [WKT notation](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) a list of 2D points representing the polygon edges `POLYGON((x1 y1, x2 y2, ...)` separated by a comma.
```

Comment (oshadmi) [inline docs/commands/ft.create.md:58]:
```suggestion
 - `GEOMETRY`- Allows polygon queries against the value in this attribute. The value of the attribute must follow [WKT notation](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) a list of 2D points representing the polygon edges `POLYGON((x1 y1, x2 y2, ...)` separated by a comma.
```

Comment (oshadmi) [reply docs/commands/ft.create.md:58]: Current limitations for `GEOMETRY`: supports only POLYGON, does not support `SORTABLE` and JSON multi values.

**Comment by rafie** [comment]
@adrianoamaral Note that this PR contains (probably by mistake) change that are not docs -> converted to draft to avoid accidental merge.

**Pull request merged**